huaweicloud/services/lts/resource_huaweicloud_lts_cce_access.go

```diff
@@ -0,0 +1,534 @@
+	}
+
+	if d.HasChanges(updateCceAccessConfigChanges...) {
+		var (
+			updateCceAccessConfigHttpUrl = "v3/{project_id}/lts/access-config"
+			updateCceAccessConfigProduct = "lts"
+		)
+		ltsClient, err := cfg.NewServiceClient(updateCceAccessConfigProduct, region)
+		if err != nil {
+			return diag.Errorf("error creating LTS client: %s", err)
+		}
+
+		updateCceAccessConfigPath := ltsClient.Endpoint + updateCceAccessConfigHttpUrl
+		updateCceAccessConfigPath = strings.ReplaceAll(updateCceAccessConfigPath, "{project_id}", ltsClient.ProjectID)
+
+		updateCceAccessConfigOpt := golangsdk.RequestOpts{
+			KeepResponseBody: true,
+		}
+
+		updateCceAccessConfigOpt.JSONBody = utils.RemoveNil(buildUpdateCceAccessConfigBodyParams(d))
+		_, err = ltsClient.Request("PUT", updateCceAccessConfigPath, &updateCceAccessConfigOpt)
+		if err != nil {
+			return diag.Errorf("error updating cce access config: %s", err)
+		}
+	}
+	return resourceCceAccessConfigRead(ctx, d, meta)
+}
+
+func buildUpdateCceAccessConfigBodyParams(d *schema.ResourceData) map[string]interface{} {
+	bodyParams := map[string]interface{}{
+		"access_config_id":     d.Id(),
+		"access_config_detail": buildCceAccessConfigDeatilRequestBody(d.Get("access_config")),
+		"host_group_info":      buildHostGroupInfoRequestBody(d),
+		"access_config_tag":    utils.ExpandResourceTags(d.Get("tags").(map[string]interface{})),
+		"cluster_id":           d.Get("cluster_id"),
+		"binary_collect":       utils.ValueIngoreEmpty(d.Get("binary_collect")),
+		"log_split":            utils.ValueIngoreEmpty(d.Get("log_split")),
+	}
+	return bodyParams
+}
+
+func cceAccessConfigResourceImportState(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	name := d.Id()
+	d.Set("name", name)
+
+	var (
+		cfg                        = meta.(*config.Config)
+		region                     = cfg.GetRegion(d)
+		listCceAccessConfigProduct = "lts"
+	)
+	ltsClient, err := cfg.NewServiceClient(listCceAccessConfigProduct, region)
+	if err != nil {
+		return []*schema.ResourceData{d}, fmt.Errorf("error creating LTS client: %s", err)
+	}
+
+	return []*schema.ResourceData{d}, refreshCceAccessID(ltsClient, d)
+}
+
+func refreshCceAccessID(ltsClient *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	var (
+		listCceAccessConfigHttpUrl = "v3/{project_id}/lts/access-config-list"
+	)
+	listCceAccessConfigPath := ltsClient.Endpoint + listCceAccessConfigHttpUrl
+	listCceAccessConfigPath = strings.ReplaceAll(listCceAccessConfigPath, "{project_id}", ltsClient.ProjectID)
+
+	listCceAccessConfigOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	name := d.Get("name").(string)
+	listCceAccessConfigOpt.JSONBody = map[string]interface{}{
+		"access_config_name_list": []string{name},
+	}
+	listCceAccessConfigResp, err := ltsClient.Request("POST", listCceAccessConfigPath, &listCceAccessConfigOpt)
+	if err != nil {
+		return fmt.Errorf("error retrieving cce access config: %s", err)
```

Comment:
CCE, not cce, please check the other places

Reply:
done